lib/processor/arm/cpu.h

```diff
@@ -14,4 +14,32 @@
 
 #define metal_cpu_yield()
 
+static inline void metal_processor_io_write64(atomic_ullong *ptr,
+					      uint64_t value,
+					      memory_order order)
+{
+  char *tmp = (char *)&value;
+  char *lptr = (char *)ptr;
+
+  atomic_store_explicit((atomic_ulong *)lptr, *((atomic_ulong *)tmp), order);
+  tmp += sizeof(atomic_ulong);
+  lptr += sizeof(atomic_ulong);
+  atomic_store_explicit((atomic_ulong *)lptr, *((atomic_ulong *)tmp), order);
+}
+
+static inline uint64_t metal_processor_io_read64(atomic_ullong *ptr,
+						 memory_order order)
+{
```

Comment:
The same as the previous function, can we use static lock in this function to make read atomic?